test/staking/Integration.ts

```diff
@@ -626,25 +649,22 @@ describe("Integration", function () {
       let currentEpoch = (await clusterSelectors[0].getCurrentEpoch()).toNumber();
       let currentPlusOne = BN.from(currentEpoch).add(1).toString();
       await clusterSelectors[0].selectClusters();
+
+      // skipping only for issuing the rewards
       await skipEpoch();
+      await skipEpoch();
+      await skipEpoch();
```

Comment:
What is the reason for skipping 3 epochs ?

Reply:
In most cases where `skipEpoch` is called multiple times, we wan't to go far ahead in time. It could be any number more than that..